sdk/src/types/block/slot/commitment.rs

```diff
@@ -31,21 +34,28 @@ pub struct SlotCommitment {
     /// the switching of a chain.
     #[serde(with = "string")]
     cumulative_weight: u64,
+    /// Reference Mana Cost (RMC) to be used in the slot with index at `index + Max Committable Age`.
+    #[serde(with = "string")]
+    reference_mana_cost: u64,
```

Comment:
It's not yet been added to the HTTP API but knowing them, it'll probably be `RMC` as JSON. We can update later, just mentioning it.

Reply:
I added a comment about the naming of this field too, because I think it should be the full name.